### PR TITLE
Trigon pid controller

### PR DESCRIPTION
**Pull request by OhadLevine**

### CODE REVIEW

**Comment by OhadLevine** [comment]
[TrigonPIDController](https://trello.com/c/ExuVK62N/117-trigonpidcontroller)

File: src/main/java/frc/robot/utils/TrigonPIDController.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package frc.robot.utils;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpiutil.math.MathUtil;
+
+public class TrigonPIDController extends PIDController {
+	/**
+	 * This is TRIGONS PIDController class with a friendlier user interface for the
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/utils/TrigonPIDController.java]: interface - don't use that word

File: src/main/java/frc/robot/utils/TrigonPIDController.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	}
+
+	/**
+	 * This method calculates the pid in a case that the sepoint is changes.
+	 * 
+	 * @param measurement The current measurement of the process variable.
+	 * @param setpoint    The new setpoint of the controller.
+	 */
+	public double trigonCalculate(double measurement, double setpoint) {
+		return MathUtil.clamp(calculate(measurement, setpoint), -1, 1);
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/utils/TrigonPIDController.java]: return MathUtil.clamp(calculate(measurement, setpoint), -1, 1);

and what if we don't want range of -1 to 1 like in the LoaderPID command?

**Review by HilelAmiel — CHANGES_REQUESTED** [review]
two things

File: src/main/java/frc/robot/utils/TrigonPIDController.java (hunk 2 above)
Comment (EliyaFishman) [reply src/main/java/frc/robot/utils/TrigonPIDController.java]: call the method calculate and not trigonCalculate

File: src/main/java/frc/robot/utils/TrigonPIDController.java (hunk 1 above)
Comment (EliyaFishman) [reply src/main/java/frc/robot/utils/TrigonPIDController.java]: change TRIGONS to TRIGON's or something similar that makes sense

**Review by EliyaFishman — CHANGES_REQUESTED** [review]
Please make the code work well

**Review by EliyaFishman — APPROVED** [review]

**Review by HilelAmiel — APPROVED** [review]

**Pull request merged**